patches/tModLoader/Terraria/Audio/SoundPlayer.cs.patch

```diff
@@ -59,10 +59,11 @@
 +				}
 +			}
 +
-+			// Handle 'UsesMusicPitch'.. This property is a weird solution, and is currently internal.
 +			SoundStyle styleCopy = style;
 +			
++			// Handle 'UsesMusicPitch' (currently used solely to deal with vanilla's older instrument items).
 +			if (style.UsesMusicPitch) {
++				styleCopy.Volume *= 0.75f;
```

Comment:
Volume multiplication is out of place, should be done in styles instead.

Reply:
I mean, this is a replication of the vanilla rule for the instrument sounds that use this property, but w/e